flocker/provision/_libcloud.py

```diff
@@ -197,153 +195,6 @@ def get_ssh_key(self):
             # https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_KeyPairInfo.html
             return None
 
-    def create_node(self, name, distribution, metadata={}):
-        """
-        Create a node.  If at first this does not succeed, try, try again.
-
-        :param str name: The name of the node.
-        :param str distribution: The name of the distribution to install on the
-            node.
-        :param dict metadata: Metadata to associate with the node.
-
-        :return libcloud.compute.base.Node: The created node.
-        """
-        size = self._default_size
-
-        image_name = self._image_names[distribution]
-
-        create_node_arguments = self._create_node_arguments()
-
-        node, addresses = self._create_with_retry(
-            name=name,
```

Comment:
in create_nodes this just logs the instance_count rather than the all of the names. Probably okay.